### PR TITLE
Scanning sonar integration

### PR DESCRIPTION
**Pull request by romulogcerqueira**
create a script to test scanning sonar

### CODE REVIEW

**Comment by doudou** [comment]
Try to remove 0f745ac ... It is an old commit by gustavo that has been deprecated in favor of another method.

**Comment by romulogcerqueira** [comment]
Done! @doudou

**Comment by doudou** [comment]
simulation/orogen/imaging_sonar_simulation does not seem to be defined in the package sets, or did I miss something ?

**Comment by romulogcerqueira** [comment]
We forgot to open a pull request about this.

File: manifest.xml [hunk 1]
```diff
@@ -10,4 +10,9 @@
   <depend package="bundles/rock" />
   <depend package="bundles/rock" />
   <depend package="gui/robot_model" />
+  <depend package="gui/vizkit3d_world" />
+  <depend package="gui/orogen/vizkit3d_world" />
```
Comment (doudou) [inline manifest.xml]: The bundle does not depend on the vizkit3d_world. Only on the simulation packages. Only add direct dependencies.

Comment (romulogcerqueira) [inline manifest.xml]: You are right. Removed vizkit3d_world dependences in manifest.xml.

**Pull request closed without merging**